docs/Plugins/zentao.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Zentao"
+description: >
+  Zentao Plugin
+---
+
+## Summary
+This plugin collects Zentao data through Zentao REST API. 
```

Comment:
Text change:
This plugin collects Zentao data through its REST APIs. [Zentao](link to its GitHub page) is an issue-tracking tool similar to Jira.